style/ruby/.spec_rubocop.yml

```diff
@@ -0,0 +1,8 @@
+inherit_from:
+  - ../.rubocop.yml
+
+Metrics/BlockLength:
+  Enabled: false
+
+Metrics/CyclomaticComplexity:
+  Enabled: false
```

Comment:
Both of these should be enabled. Block length prevents people from writing unwieldy, overly-complicated blocks. Cyclomatic complexity prevents people from writing overly-complicated, excessively-clever or dense code. In fact, CC is one of the single most valuable measurements of code quality that CodeClimate provides.

Reply:
Just noticed that these are for specs? I'm unclear on how this file would be used in our current spec suite.